docs/pages/versions/unversioned/sdk/expo.mdx

```diff
@@ -66,11 +84,58 @@ reader.read().then(({ done, value }) => {
 });
 ```
 
+### `URL` API
+
+`URL` provides the standard API on all platforms.
+
+On native platforms, built-in `URL` and `URLSearchParams` implementations replace the shims in `react-native`. Refer to the [browser and server runtime support](https://caniuse.com/url) for web and Node.js.
+
+```ts URL and URLSearchParams
+const url = new URL('https://expo.dev');
+
+const params = new URLSearchParams();
+```
+
+Expo's built-in `URL` support attempts to be fully [spec compliant](https://developer.mozilla.org/en-US/docs/Web/API/URL). The only missing exception is that native platforms do not currently support [non-ASCII characters](https://unicode.org/reports/tr46/) in the hostname.
+
+```ts Non-ASCII characters
+console.log(new URL('http://🥓').toString());
+
+// This outputs the following:
+// - Web, Node.js: http://xn--pr9h/
+// - Android, iOS: http://🥓/
+```
+
 <APISection packageName="expo" />
 
 ## Common questions
 
-### What if I want to name my main app file something other than App.js or app/\_layout.tsx?
+Some common questions about using the `expo` package in your project.
+
+### registerRootComponent
+
```

Comment:
```suggestion
```

This header seems redundant, for general usage we should refer to autogenerated header from API - http://docs.expo.dev-pr-36443.s3-website-us-east-1.amazonaws.com/versions/unversioned/sdk/expo/#registerrootcomponentcomponent

Reply:
Makes sense. The idea was to allow more common questions per API/method but I guess we do not have more common questions right now. Maybe we can revisit this in future.